UnitsNet.Tests/CustomCode/SpeedTests.cs

```diff
@@ -68,6 +68,12 @@ public class SpeedTests : SpeedTestsBase
 
         protected override double MillimetersPerHourInOneMeterPerSecond => 3600000;
 
+        protected override double InchesPerSecondInOneMeterPerSecond => 39.3700787;
```

Comment:
I would recommend googling for some guides on git, github, forks and pull requests. I haven't read this one, but at first glance it seems to cover the basics: https://gist.github.com/Chaser324/ce0505fbed06b947d962

The basic gist of it is:
* Add official repo as remote so you can sync in changes, I prefer to call this `origin` and if I have a fork I call that `my`, but that is just my preference: `git remote set-url origin https://github.com/anjdreas/unitsnet` and `git remote add my https://github.com/gojanpaolo/unitsnet`
* Make sure your local `master` branch points to `origin/master` (official one), since I don't think it makes sense to use your fork's master for much: `git branch --set-upstream-to=origin/master`
* Checkout `master` and pull in the latest changes from `origin`: `git checkout master && git pull --rebase`
* Create a local branch (based on `origin/master`) for any new work you want to create a pull request for, `fix-unit-parsing` or `add-pascal-unit`: `git checkout -b fix-unit-parsing origin/master`
* Do your work as one or more commits
* When you are ready for review, `rebase` your branch on top of any new commits on the official repo, then push that branch up to your github fork: `git rebase origin/master && git push -u my`
* Visit github.com to create a pull request from that branch

It's a bit boilerplate the first time, especially setting up the remotes and stuff, but it gets easier with practice. You might also want to use some GUI clients to make things easier, but I personally prefer the command line.